<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简单麦麦客户端-www.jiandanmaimai.cn</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="2" column="0">
    <widget class="QLabel" name="message_label">
     <property name="text">
      <string>信息提示栏</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="mainTab">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="loginTab">
      <attribute name="title">
       <string>用户登录</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QFrame" name="stateFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="1">
             <widget class="QLabel" name="userNameLabel">
              <property name="text">
               <string>未登录</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>用户名：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>当前状态：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="stateLable">
              <property name="text">
               <string>未登录</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>用户ID：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="uidLabel">
              <property name="text">
               <string>未登录</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFrame" name="loginFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>登录简单麦麦</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>用户账号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="userNameEdit">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>密码</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pwdEdit">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="loginBtn">
              <property name="text">
               <string>登录</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>如果没有账号密码 可以在【简单麦麦】公众号回复 登录失败 获取</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>331</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="uploadTab">
      <attribute name="title">
       <string>网盘链接批量上传</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="insertBtn">
         <property name="text">
          <string>增加行</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="deleteBtn">
         <property name="text">
          <string>删除当前行</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTableWidget" name="inforTable">
         <column>
          <property name="text">
           <string>提示信息</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>标题</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>描述</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>网盘链接</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>网盘提取码</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>解压密码</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>价格</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>文件类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>文件大小</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="cateCb"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="publishBtn">
         <property name="text">
          <string>发布</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="clearBtn">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="localServerTab">
      <attribute name="title">
       <string>本地分享服务器管理中心</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;注意：此种方式上传文件需保持电脑开机且程序运行 才可以被他人方便地下载 &lt;a href=&quot;http://www.jiandanmaimai.cn/blog/article/8/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;首次使用请点此&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>新发布文件区域</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>设置已有文件下载方式</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>标题</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="client_title_input">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>文件ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="file_id_input">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="check_file_btn">
           <property name="text">
            <string>检测文件可用性</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>描述</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>文件标题：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="file_title_label">
           <property name="text">
            <string>文件标题</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" rowspan="7">
        <widget class="QPlainTextEdit" name="client_des_input"/>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="client_choose_files_btn_2">
           <property name="text">
            <string>选择需要设置的文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="set_file_download_btn">
           <property name="text">
            <string>设置该文件的下载方式</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>当前客户端ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="client_id_label">
           <property name="text">
            <string>-1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>当前客户端访问地址</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="client_ipv6_label">
           <property name="text">
            <string>-1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>当前MAC地址</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="client_mac_label">
           <property name="text">
            <string>-1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPushButton" name="client_clear_log_btn">
           <property name="text">
            <string>清空所有日志</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="client_registe_client_btn">
           <property name="text">
            <string>注册该客户端</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>日志框</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1" rowspan="6">
        <widget class="QTextBrowser" name="client_log_text"/>
       </item>
       <item row="11" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>价格   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="client_money_input">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="12" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>文件解压密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="client_unpress_pwd_input">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="13" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>文件分类</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="client_cate_cb"/>
         </item>
        </layout>
       </item>
       <item row="14" column="0">
        <widget class="QPushButton" name="client_choose_files_btn">
         <property name="text">
          <string>选择需要销售的文件</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QPushButton" name="client_upload_btn">
         <property name="text">
          <string>开始上传并处理文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="accountInfoTab">
      <attribute name="title">
       <string>账号管理页</string>
      </attribute>
      <widget class="QPushButton" name="generate_file_xls_btn">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>150</y>
         <width>231</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>导出当前所有文件的xls信息</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>380</y>
         <width>921</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>调试信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="account_info_log_text"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
